### ISSUE TITLE
[Card - pictogram] Web Component: Change to include mobile accessibility/hover state improvements - DDS consulting

### ISSUE THREAD

**Issue by RobertaJHahn** [carbon-design-system/carbon-for-ibm-dotcom#7553]
###  Engineering info:
- HC engineer: @proeung 
- HC Jira Ticket: https://jsw.ibm.com/browse/HC-2328
- DDS consulting engineer: Pending DDS resourcing review
- DDS Designer: Saijan Kuruvila 
- DDS engineering sizing:  5

----

#### User Story

> As a `[user role below]`:
Carbon for ibm.com developer

> I need to:
create/change the `Card - pictogram`

> so that I can:
provide the ibm.com adopter developers components they can use to build ibm.com web pages

#### Additional information

- Story within Storybook with corresponding knobs
- Utilize Carbon
- Create with Shadow DOM and Custom Elements standards
- **See the Epic (#6739 ) for the Design and Functional specs information**
- [Design and func specs Box folder](https://ibm.ent.box.com/folder/95514001950?s=hgvu48r1fv2ga2h0yn3qshrtxjc71vch)
- This work includes the **animation** change on the desktop, motion specs are included in the Box folder, [specific subfolder link here](https://ibm.box.com/s/ykv2ukr3twyfx2nrro3rn7ge4iqoz48x).  
- Web Components Visual QA testing issue (#7554 )
- Web Components Prod QA testing issue (#7555 )

#### Acceptance criteria
- [ ] Include README for the web component and corresponding styles
- [ ] Create Web Components styles in styles package
- [ ] Do not create knobs in Storybook that include JSON objects
- [ ] Break out Storybook stories into multiple variation stories, if applicable
- [ ] Create codesandbox example under `/packages/web-components/examples/codesandbox` and include in README
- [ ] Minimum 80% unit test coverage
- [ ] Update the Carbon for ibm.com website component [data file](https://github.com/carbon-design-system/carbon-for-ibm-dotcom-website/blob/master/src/data/components.json) to be sure the web site Component Status and Storybook links are correct
- [ ] Use the [Visual QA checklist](https://github.com/carbon-design-system/carbon-for-ibm-dotcom/wiki/Definition-of-done-(Visual-QA-checklist)) to verify design quality
- [ ] If a design is provided, the Designer is included as a Reviewer in the Pull Request
- [ ] Provide a direct link to the deploy preview for the designer in the Pull Request description
- [ ] A comment is posted in the Prod QA issue, tagging Praveen when development is finished

**Comment by shixiedesign** [comment]
Links should be updated in both this issue and epic. Ready for dev to pick up. 
Design spec: https://ibm.ent.box.com/folder/114042319288
Functional spec: https://ibm.box.com/s/hgvu48r1fv2ga2h0yn3qshrtxjc71vch

**Comment by proeung** [comment]
See the Cards pictograms bottom and top specs for the treatment on desktop and mobile - https://ibm.ent.box.com/file/869349975806
